src/libs/UserUtils.ts

```diff
@@ -104,7 +104,7 @@ function getDefaultAvatarURL(accountID: string | number = '', isNewDot = false):
  * Given a user's avatar path, returns true if user doesn't have an avatar or if URL points to a default avatar
  * @param [avatarURL] - the avatar source from user's personalDetails
  */
-function isDefaultAvatar(avatarURL?: string): boolean {
+function isDefaultAvatar(avatarURL?: string | React.FC<SvgProps>): boolean {
```

Comment:
@kubabutkiewicz maybe you can align it with Fabio updates in [this PR](https://github.com/Expensify/App/pull/30328/files#diff-6bde6d64d118fa3586ce779a52e1fd38220d561bc4ec71cc60881f812a4b9d94R11) to escape conflicts in the future